build.sbt

```diff
@@ -255,6 +262,10 @@ lazy val `simulator-gui-new` = project
   .settings(
     name := "simulator-gui-new",
     libraryDependencies ++= Seq(scopt, scalatest.value, scalaFX.value) ++ javaFXBinary.value,
+    assembly / assemblyMergeStrategy := {
```

Comment:
does it need to be repeated here?

Reply:
I leaved that to make it possible to assemble only the simulator-gui-new submodule... it makes sense?